### PR TITLE
Add examples of custom plugins

### PR DESCRIPTION
**Pull request by Edouard2laire**
Linked to https://github.com/brainstorm-tools/brainstorm3/pull/683, this add two example of user-defined plugin we could use for the tutorial

### CODE REVIEW

File: plugins/plugin_bids_matlab.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Name": "bids_matlab",
+  "Version": "latest",
+  "Category": "Other",
+  "URLzip": "https://github.com/bids-standard/bids-matlab/archive/refs/heads/main.zip",
+  "URLinfo": "https://github.com/bids-standard/bids-matlab",
+  "TestFile": "+bids/bids_matlab_version.m",
```
Comment (rcassani) [inline plugins/plugin_bids_matlab.json]: `+`, typo?

File: plugins/plugin_bids_matlab.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "Name": "bids_matlab",
+  "Version": "latest",
+  "Category": "Other",
```
Comment (rcassani) [inline plugins/plugin_bids_matlab.json]: Since `Category` is overridden, we should remove it

File: plugins/plugin_bst_users.json
Comment (rcassani) [inline plugins/plugin_bst_users.json:1]: I think this file makes more sense in the root of the repo

File: plugins/plugin_bids_matlab.json
Comment (rcassani) [inline plugins/plugin_bids_matlab.json:1]: What do you think of having this file in an explicit dir `user_def_plugins`

File: plugins/plugin_bst_users.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "Name": "bst-users",
+  "Version": "latest",
+  "Category": "Other",
+  "URLzip": "https://github.com/brainstorm-tools/bst-users/archive/refs/heads/master.zip",
+  "URLinfo": "https://github.com/brainstorm-tools/bst-users/",
+  "TestFile": "processes/examples/process_beamformer_test.m",
+  "LoadFolders": [
```
Comment (rcassani) [inline plugins/plugin_bst_users.json]: Load only the processes, not the scripts

File: plugins/plugin_bids_matlab.json (hunk 1 above)
Comment (Edouard2laire) [reply plugins/plugin_bids_matlab.json]: no there is really a + : https://github.com/bids-standard/bids-matlab/blob/main/%2Bbids/bids_matlab_version.m

File: plugins/plugin_bids_matlab.json (hunk 2 above)
Comment (Edouard2laire) [reply plugins/plugin_bids_matlab.json]: Done

File: plugins/plugin_bids_matlab.json
Comment (Edouard2laire) [reply plugins/plugin_bids_matlab.json:1]: Done

**Comment by Edouard2laire** [comment]
ok; this is ready :)

**Comment by rcassani** [comment]
Thanks for performing the changes

**Pull request merged**